src/pages/postgraphile/usage-library.md

```diff
@@ -137,6 +137,24 @@ create function get_x_something() returns text as $$
 $$ language sql stable;
 ```
 
+Everything returned by pgSettings is applied to the current session with ```set_config($key, $value, true)```
```

Comment:
It would be good to have a note here stating that the values should all be strings, since `set_config` only accepts strings, and that it's up to your application to convert them back to whatever type they need.